app/views/orders/_charts.html.erb

```diff
@@ -0,0 +1,89 @@
+<div class="content-wrapper">
+  <% flash.each do |key, value| %>
+    <%= content_tag :div, value, class: "col-6 offset-3 alert alert-#{key}" %>
+  <% end %>
+    <div class="container-fluid">
+      <!-- Icon Cards-->
+      <div class="row">
+        <div class="col-xl-3 col-sm-6 mb-3">
+          <div class="card text-white bg-success o-hidden h-100">
+            <div class="card-body">
+              <div class="card-body-icon">
+                <i class="fa fa-fw fa-shopping-cart"></i>
+              </div>
+              <div class="mr-5">There are <%=Order.count%> Orders!</div>
+            </div>
+            <a class="card-footer text-white clearfix small z-1" href="#">
+              <span class="float-left">View Details</span>
+              <span class="float-right">
+                <i class="fa fa-angle-right"></i>
+              </span>
+            </a>
+          </div>
+        </div>
+        <div class="col-xl-3 col-sm-6 mb-3">
+          <div class="card text-white bg-danger o-hidden h-100">
+            <div class="card-body">
+              <div class="card-body-icon">
+                <i class="fa fa-fw fa-support"></i>
+              </div>
+              <div class="mr-5"><%=@orders.where(status: 'pending').count%> Pending Orders!</div>
+            </div>
```

Comment:
one of the requirements is to use haml for the front end :-)

Reply:
Done, changed views. Sorry, not used to using haml